src/filter/GaussianFilter.js

```diff
@@ -0,0 +1,58 @@
+import IJ from '../ij';
+
+export default function gaussian_filter(n, sigma){
+	var cols = this.width;
+	var new_img = this.clone();
+	var kernel = get_gaussian_kernel(n, sigma);
+	var temp = cols*n; //variable for checking the image border
+	for(var p = cols*n+n; p < this.data.length - cols*n; p++){
+		if(p == temp+cols-n){
+			p += 2*n - 1;
+		}else{
+			new_img.data[p] = convolution(p, this.data, kernel);
```

Comment:
In this project we make the hypothesis that a filter change the original image "in place". This means that in this filter it is "this" that should change. This means that in this case we should have `this.data[p] = convolution(p, new_img.data, kernel);`